skylib/k8s.bzl

```diff
@@ -76,6 +76,7 @@ show = rule(
 def k8s_deploy(
         name,  # name of the rule is important for gitops, since it will become a part of the target manifest file name in /cloud
         cluster = "dev",
+        user = "{BUILD_USER}",
```

Comment:
We have per parameter table in https://github.com/adobe/rules_gitops/blob/master/README.md please add a row about user